### ISSUE TITLE
How to build the project?

### ISSUE THREAD

**Issue by Himujjal** [chwayne/rem#1]
Running `zig build` gives the following error:

```
error: unable to load '/**/**/rem/tools/named_characters_data.zig': FileNotFound
const named_characters_data = @import("named-characters-data");
```

Zig version: `0.9.1`

**Comment by chwayne** [comment]
Run this command: `zig build gen-named-characters-data`

I'll make a commit that does this automatically soon.

**Comment by chwayne** [comment]
Fixed in d5478c0fd297df9f62bab8c759503a6c221b9673
